dbux-code/src/commands/runCommands.js

```diff
@@ -46,6 +46,9 @@ function getArgs(debugMode) {
   let env = config.get(`dbux.${runMode}.env`);
   dbuxArgs += `${parseEnv(env)}`;
 
+  let packageWhitelists = config.get(`dbux.packageWhitelist`);
+  if (packageWhitelists) dbuxArgs += packageWhitelists.split(',').map(packageName => ` -w ${packageName}`).join(' ');
```

Comment:
Why change the format of `packageWhitelists` from space-separated to comma-separated. Can we keep it the same (space-separated, is it?) everywhere?